components/pages/aunthentication/auth.css

```diff
@@ -0,0 +1,46 @@
+.form-container{
+    display: flex;
+    width: 100%;
+    height:100vh;
+    align-items: center;
+    justify-content: center;
+}
+
+.form{
+    background-color: var(--bg-color);
+    width:35rem;
+    height:fit-content;
+    align-items: center;
+    justify-content: center;
+    font-weight: 600;
+}
+
+.form-heading{
+    color: var(--primary-color);
+    font-family: 'Times New Roman', Times, serif;
+}
+
+
+.form>input{
+    margin: 1.5rem;
+    padding:1rem;
+    width: 25rem;
+    height: 3rem;
+    border: 2px solid var(--primary-color);
+}
+
+.btn{
+    font-size: large;
+    background-color: var(--primary-color);
+    margin: 1.5rem;
+    width: 8rem;
+    color: black;
+}
+
+.sign-up{
+    color:tomato;
```

Comment:
Use color variable

Reply:
Is it necessary to use variable if I'm going to use the color only once?